src/main/java/Models/TrainingSystem.java

```diff
@@ -0,0 +1,42 @@
+package Models;
+
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.*;
+
+public class TrainingSystem {
+
+    public static void main(String[] args) {
+        List<Course> javaCourses = new ArrayList<>();
+        List<Course> testingCourses = new ArrayList<>();
+        Map<Student, TrainingProgram> studentCourses = new HashMap<>();
+
+        javaCourses.add(new Course("Java Basic", 3));
+        javaCourses.add(new Course("Java Core", 1));
+        javaCourses.add(new Course("Testing", 5));
+
+        testingCourses.add(new Course("Basics of testing", 5));
+        testingCourses.add(new Course("Manual testing vs Automation testing", 10));
+        testingCourses.add(new Course("Selenium", 8));
+
+        try{
+            TrainingProgram javaTraining = new TrainingProgram("Java", javaCourses, LocalDateTime.of(2021,12,30,10,0));
+            TrainingProgram testingProgram = new TrainingProgram("Testing", testingCourses, LocalDateTime.of(2021,12,30,10,0));
+
+            studentCourses.put(new Student("Ivanov", "Ivan"), javaTraining);
+            studentCourses.put(new Student("Sidorov", "Ivan"), testingProgram);
+        }
+        catch (DateTimeException ex){
+            ex.printStackTrace();
+            System.exit(1);
```

Comment:
instead of manually exiting the program, please put code from lines 34-40 inside the try block